themes/digital.gov/src/scss/new/styles.scss

```diff
@@ -162,7 +162,6 @@
 @forward 'list-topics';
 
 @forward 'breadcrumbs';
-@forward 'clicks';
```

Comment:
We can remove click styles on cards too

https://github.com/GSA/digitalgov.gov/blob/23a07f637260ca0c1167bc83bddaa06eddba67ff/themes/digital.gov/src/scss/new/_card.scss#L75-L89

Reply:
Removed in [65e10a8](https://github.com/GSA/digitalgov.gov/pull/5310/commits/65e10a80365b58d02d65fce754210bc71b46f4d0)